scene/gui/item_list.cpp

```diff
@@ -295,35 +295,19 @@ int ItemList::get_current() const {
 	return current;
 }
 
-void ItemList::move_item(int p_item, int p_to_pos) {
+void ItemList::move_item(int from_idx, int to_idx) {
```

Comment:
Should be prefixed with `p_` anyway, as that's how most/all functions' parameters in the core are listed.